.gitignore

```diff
@@ -5,3 +5,4 @@ venv/
 
 .idea/
 .vscode/
+.DS_Store
```

Comment:
I'll allow it, just because we're already have IDE specifics in the ignore file.